src/sources/host_metrics/mod.rs

```diff
@@ -32,33 +33,34 @@ mod memory;
 mod network;
 
 /// Collector types.
+#[serde_as]
 #[configurable_component]
 #[derive(Clone, Copy, Debug, Eq, PartialEq)]
 #[serde(rename_all = "lowercase")]
 pub enum Collector {
-    /// CGroups.
+    /// Metrics related to Linux control groups.
     #[cfg(target_os = "linux")]
     CGroups,
 
-    /// CPU.
+    /// Metrics related to CPU utilization.
     Cpu,
 
-    /// Disk.
+    /// Metrics related to disk I/O utilization.
     Disk,
 
-    /// Filesystem.
+    /// Metrics related to filesystem space utilization.
     Filesystem,
 
-    /// Load average.
+    /// Load average metrics.
```

Comment:
nit- any reason not to word as "Metrics related to load average" ? (for consistency)

Reply:
I was porting over language in the old documentation, but I'll make this consistent.